client/packages/coldchain/src/Monitoring/ListView/TemperatureBreach/TemperatureBreachList.tsx

```diff
@@ -98,12 +102,19 @@ const ListView: FC = () => {
     [
       {
         key: 'acknowledgedIcon',
+        sortable: false,
```

Comment:
offroad.. noticed that you could try to sort by the image column, which of course fails.